package.json

```diff
@@ -13,6 +13,7 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.17.9",
+    "@chakra-ui/cli": "0.0.0-pr-20211126153854",
```

Comment:
Why we use this version of `@chakra-ui/cli`? It seems that the latest one is `2.1.0` but ideally I think we should use the same that we use in token-dashbaord which is `^1.9.0`

Reply:
It's a stupid bug with mac silicon chips: https://github.com/chakra-ui/chakra-ui/issues/5219#issuecomment-1001203749